test/test_deprecations.jl

```diff
@@ -122,4 +122,8 @@ end
     @test_throws ErrorException(msg) Ferrite.getfielddim(dh.subdofhandlers[1], :u)
 end
 
+@testset "default_interpolation" begin
+    @test Ferrite.default_interpolation(Triangle) == geometric_interpolation(Triangle)
```

Comment:
```suggestion
    @test Ferrite.default_interpolation(Triangle) == geometric_interpolation(Triangle)
    @test Ferrite.default_interpolation(Triangle((1,2,3))) == geometric_interpolation(Triangle((1,2,3)))
```
To have a regression for this.

Reply:
That never worked AFAIU (verified by checking out the commit before merging #953:  572c1068c7ed38720ef54303d97c8a829263be57)